_topic_map.yml

```diff
@@ -1275,23 +1275,23 @@ Name: Monitoring
 Dir: monitoring
 Distros: openshift-enterprise,openshift-webscale,openshift-origin
 Topics:
-- Name: Cluster monitoring
-  Dir: cluster_monitoring
-  Topics:
-  - Name: About cluster monitoring
-    File: about-cluster-monitoring
-  - Name: Configuring the monitoring stack
-    File: configuring-the-monitoring-stack
-  - Name: Managing cluster alerts
-    File: managing-cluster-alerts
-  - Name: Examining cluster metrics
-    File: examining-cluster-metrics
-  - Name: Accessing Prometheus, Alertmanager, and Grafana
-    File: prometheus-alertmanager-and-grafana
-- Name: Monitoring your own services
-  File: monitoring-your-own-services
+- Name: Understanding monitoring
+  File: understanding-monitoring
+- Name: Configuring platform monitoring
+  File: configuring-platform-monitoring
+- Name: Managing application metrics
```

Comment:
Already explained in https://github.com/openshift/openshift-docs/pull/24722#discussion_r472039388

Reply:
Does 'Managing metrics for your own workloads' sound suitable? This provides clarity around the type of metrics being managed, but it is less likely to sound like developer applications only.

Reply:
That's fine.

Reply:
I have now changed this title to 'Managing metrics for your own workloads'.